### ISSUE TITLE
changes stream stop update for 4 hours

### ISSUE THREAD

**Issue by fengmk2** [npm/registry-issue-archive#129]
```bash
curl -v 'https://replicate.npmjs.com/_changes?include_docs=false&since=1006693'
```

last seq is 1006694, and never update after that.

**Comment by fengmk2** [comment]
Fixed by @soldair https://twitter.com/soldair/status/833633394948968449

**Comment by fengmk2** [comment]
replicate delay again

https://replicate.npmjs.com/antd-tools/

![image](https://cloud.githubusercontent.com/assets/156269/24240658/641081ba-0fed-11e7-8c67-0e6819114622.png)

but registry is 0.19.0

![image](https://cloud.githubusercontent.com/assets/156269/24240667/71f3618a-0fed-11e7-90f1-a00485892e31.png)

**Comment by fengmk2** [comment]
ping @soldair

**Comment by fengmk2** [comment]
And API response 502 now: https://registry.npmjs.org/-/all/since?stale=update_after&startkey=1490251801073

![image](https://cloud.githubusercontent.com/assets/156269/24240752/a8f90d2e-0fed-11e7-9258-d1240194c1d4.png)

**Comment by afc163** [comment]
Run into this too.

**Comment by soldair** [comment]
taking a look now

**Comment by soldair** [comment]
seems to be resolved now `"latest" : "0.19.4"`
"update_seq":1225066

**Comment by fengmk2** [comment]
@soldair replicate is ok now
